### PR TITLE
Fix the watchtower pre-check random wait

### PR DESCRIPTION
**Pull request by joaosa**
I noticed this was broken when revisiting this for allowing faster updates on testnet. To fix this: 

- I added running the l1 container with `--init` so signals can be properly caught as we're running multiple processes;
- I also added a new env var to configure the order of magnitude of the random wait;
- I used a script for the random wait as that is the supported way to do this with watchtower.

### CODE REVIEW

**Comment by guanzo** [comment]
huh, so this is why deployments were rolling out so quickly

**Review by guanzo — APPROVED** [review]

**Comment by joaosa** [comment]
@guanzo @DiegoRBaquero We cannot merge this or it'll break auto-update. This is because docker-compose.yml changes make it to the mainnet as soon as they're merged. I'm going to try to find a way to make this a bit saner.

**Comment by DiegoRBaquero** [comment]
Let's merge this tomorrow or monday

**Pull request merged**
